--- a/gettext-runtime/intl/Makefile.in	2016-06-11 16:37:41.000000000 +0900
+++ b/gettext-runtime/intl/Makefile.in	2018-09-24 01:31:29.479649600 +0900
@@ -108,7 +108,7 @@
 LDFLAGS = @LDFLAGS@ $(LDFLAGS_@WOE32DLL@)
 LDFLAGS_yes = -Wl,--export-all-symbols
 LDFLAGS_no =
-LIBS = @LIBS@
+LIBS = @LIBS@ -XCClinker -static-libgcc
 
 COMPILE = $(CC) -c $(DEFS) $(INCLUDES) $(CPPFLAGS) $(CFLAGS) $(XCFLAGS)
 
